587 restyle signin

fighawkins opened the pull request:
## This PR:
Resolves #587 
 


![Screenshot 2024-03-01 at 12 20 24 PM](https://github.com/codeforpdx/PASS/assets/143238486/ea56d51f-b737-4771-879c-4bb59f9d89be)




## Additional Context (optional):
Add any other context about the PR here.

## Future Steps/PRs Needed to Finish This Work (optional):
 
I think the footer could still use to be less chonky, but would require some more thought out design, so I did not completely retool it.

fighawkins committed:
restyled signin page

fighawkins committed:
adjusted footer

andycwilliams commented:
There is already an issue (#463) and pull request (#547) for this

andycwilliams commented on src/components/Footer/Footer.jsx:
```diff
@@ -1,39 +1,30 @@
- * @memberof Footer
- * @name Footer
- * @returns {React.JSX.Element} The Footer component
- */
```
We need to keep all of these JSDocs. We're using them for the whole project and it would be unusual to remove them just for these.

andycwilliams commented on src/components/Footer/Footer.jsx:
```diff
@@ -1,39 +1,30 @@
-// React Imports
```
As with the JSDocs, we use these labels across all the components. While not essential for the app to run, it maintains consistency and improves clarity.

andycwilliams reviewed (changes requested):
I haven't had the chance to run this branch locally just yet. The screenshot looks nice enough. It's just that this is already an assigned issue and it's best to avoid duplication and risking confusion. 

Plus we should keep the JSDocs. I'm not sure if someone requested that but even if it was something we all agreed upon, it ought to be a separate issue and is outside of the scope of this current one.

Jared-Krajewski commented:
> I haven't had the chance to run this branch locally just yet. The screenshot looks nice enough. It's just that this is already an assigned issue and it's best to avoid duplication and risking confusion.
> 
> Plus we should keep the JSDocs. I'm not sure if someone requested that but even if it was something we all agreed upon, it ought to be a separate issue and is outside of the scope of this current one.

That member has gone inactive so I think its fine to let @fighawkins take over the issue and remove the stale PR. Just make sure concerns from that issue and PR are addressed.

andycwilliams commented:
> > I haven't had the chance to run this branch locally just yet. The screenshot looks nice enough. It's just that this is already an assigned issue and it's best to avoid duplication and risking confusion.
> > Plus we should keep the JSDocs. I'm not sure if someone requested that but even if it was something we all agreed upon, it ought to be a separate issue and is outside of the scope of this current one.
> 
> That member has gone inactive so I think its fine to let @fighawkins take over the issue and remove the stale PR. Just make sure concerns from that issue and PR are addressed.

I took over that PR a couple days ago.

After reassessing the open issues, we agreed to refine them so they cover different needs. So this can remain focused on styling enhancements for `Signup.jsx` (though see Tim's comment in #587 for an update). Whereas #463 now focuses on refactoring the remove image prompt to a modal.

The JSDocs and import labels should still be added back in, however.

fighawkins commented:
Hey, sorry for the confusion. I have been recovering from an accident, but will get on addressing concerns here, putting the jsdocs back in, etc. Thank you.

timbot1789 commented:
@fighawkins Any updates on this PR?

andycwilliams commented:
@fighawkins Checking in again for updates
